scripts/test.ps1

```diff
@@ -148,22 +148,28 @@ function Run-Test([string[]] $testContainers, [string[]] $netCoreTestContainers)
     {
        $additionalArguments += "/parallel"
     }
-
-	if(!(Test-Path $vstestPath))
-	{
-		Write-Error "Unable to find vstest.console.exe at $vstestPath. Test aborted."
-	}
-	
-	Write-Verbose "$vstestPath $testContainers $additionalArguments /logger:trx"
-	& $vstestPath $testContainers $additionalArguments /logger:trx
 	
-	if(!(Test-Path $TPV2VSTestPath))
+	if($testContainers.Count -gt 0)
 	{
-		Write-Error "Unable to find vstest.console.exe at $TPV2VSTestPath. Test aborted."
+		if(!(Test-Path $vstestPath))
+		{
+			Write-Error "Unable to find vstest.console.exe at $vstestPath. Test aborted."
+		}
+	
+		Write-Verbose "$vstestPath $testContainers $additionalArguments /logger:trx"
```

Comment:
TPV2 vstest.console does not work? We might just want to use that everywhere for consistency. You can take this in the next PR.